src/components/calcite-date-picker-day/calcite-date-picker-day.tsx

```diff
@@ -59,6 +59,7 @@ export class CalciteDatePickerDay {
   /** Date is the end of date range */
   @Prop({ reflect: true }) endOfRange = false;
 
+  /** Date is being hovered and within the set range */
```

Comment:
@paulcpederson @jcfranco I wonder if we should set all these props to be internal since a user would never create this component? Same with the other internal date-picker components. Can we just opt a whole file out to be internal??

Reply:
I think it's fine as long as support components don't get pulled into the SDK pages. @benelan @julio8a WDYT?

Separate discussion, but I imagine a few of these can be converted into functional components too.